File: Dasar/Go/C1_HelloWorld.go
package main

import (
	"fmt"
	"rsc.io/quote"
)

func fungsi(a, b string) (string, string){
	c:=a+b
	fmt.Println(c)
	//? c=a,b <---error
	return c, quote.Go();
}

func main() {
	a,b := fungsi("Hello","World")
	fmt.Print(a+" "+b)
}

File: Basic/Java/C1_HelloWorld.java
// package v1basic;

public class C1_HelloWorld{
    public static String fungsi(String a, String b){

        String c = a + b;
        System.out.println(a+b);
        //? c=a,b; <----- error
        return c;

    }ing 

    public static void main(String[] args) {
        // ! CETAK HELLO WORLD
        System.out.print(fungsi("Hello ","World"));
    }
}